governance.md

```diff
@@ -9,15 +9,31 @@ This document describes the governance model for the Prebid project. The Prebid
 
 ### Roles and Responsibilities:
 - **User:** Any individual who consumes / uses the Prebid.js library.
-- **Contributor:** Any individual who contributes code that is subsequently merged to the project. Contributed code is governed by the Prebid.js [license](https://github.com/prebid/Prebid.js/blob/master/LICENSE). Contributors are required to sign a CLA before any code can be committed (CLA pending).
-- **Core Team Member:** An individual contributor who has been appointed by the Tech Lead on the project to maintain it and further it’s stated goals.
+- **Contributor:** Any individual who contributes code that is subsequently merged to the project. Contributed code is governed by the Prebid.js [license](https://github.com/prebid/Prebid.js/blob/master/LICENSE).
+- **Core & Review Team Member:** An individual contributor who has been appointed by the Tech Lead on the project to maintain it and further it’s stated goals.
```

Comment:
Do we want to add anything here about prebid org membership? Do we have any reviewers not part of Prebid org?

Reply:
It appears all reviewers are members

Reply:
I'm not sure if we want to revoke core team status upon member company departure, but it hasn't come up yet afaik. Mark for example just asked to be taken off upon departure from Magnite.

Reply:
yeah I was thinking about that, I don't think we need to limit it to member companies at this time.